### DISCUSSION TITLE
No search results in full text search

### DISCUSSION THREAD

**Discussion by Dav-Git** [nextcloud/all-in-one#3672] [Questions] [answered]
Hi there, I recently moved my nextcloud-aio install onto a new device with more processing power and therefore looked forward to enabling full text search. Now full text search seems to be running and the app is installed as well. However, whenever I search for any thing, the search returns no results at all.

Is there a manual way to trigger a first indexing. It seems like the usual auto-index on first startup might not have been started. Maybe because I installed from a backup?

I have seen #1133 but that doesn't really help much.

Any help much appreciated.

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/2509 and check if that makes it work?

**Comment by Dav-Git** [comment]
That did it. Thanks :)
